easybuild/main.py

```diff
@@ -272,6 +272,13 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
         search_easyconfigs(search_query, short=options.search_short, filename_only=options.search_filename,
                            terse=options.terse)
 
+    if options.latest_self:
+        if not orig_paths:
+            eb_file = find_easybuild_eb()
+            orig_paths.append(eb_file)
+        else:
+            raise EasyBuildError("Update option isn't compatible with other easyconfig files")
```

Comment:
make this `"Installing the latest EasyBuild release can not be combined with installing other easyconfigs"`